docs/public/_redirects

```diff
@@ -177,6 +177,8 @@ https://v4.material-ui.com/* https://v4.mui.com/:splat 301!
 /fr/* /:splat 200
 /de/* /:splat 200
 /ja/* /:splat 200
+/ru/* /:splat 200
```

Comment:
These are not redirections, they are proxies.

Reply:
Right, I haven't noticed the status code. Were these previously added for the new pages only?

Reply:
Or should we actually make these redirects if we no longer want to support these languages?

Reply:
`fr`, `de`, `ja` were not server-side rendered (next.js exportPathMap function), so we had to proxied them. In our case, we can turn it to a 301. No need for the `!` as the pages won't exist.

Reply:
Aaaah ok, it makes sense now, wasn't sure why these three were only added. Thanks for the clarification. Will update them now